cmd/writefreely/main.go

```diff
@@ -85,6 +85,11 @@ func main() {
 				Usage:  "Delete a user with the given username",
 				Hidden: true,
 			},
+			&cli.StringFlag{
+				Name:   "silence-user",
```

Comment:
We kept old flags in here only for backwards compatibility. Ideally we'd move away from this older style, so we don't need to add new commands here.